packages/engine/lib/execution/src/runner/python/batch.py

```diff
@@ -241,7 +217,7 @@ def flush_changes(self, schema, skip):
                 continue  # Not supposed to have this column in `cols`?
 
             field = schema.field(i_field)
-            if field.name in any_type_fields:
+            if "any_type" in field.metadata:
```

Comment:
Do you not need to do `and field.metadata["any_type"] == "1"` here?

Reply:
No need to spend a lot of time investigating it though, just wondering if you have a sense as to why

Reply:
Makes sense and I definitely agree with your decision. Thanks!